Update automated-device-enrollment-authentication.md (#4032)
MicrosoftDocs/memdocs: memdocs/intune/enrollment/automated-device-enrollment-authentication.md

 - Automatically install the Company Portal app during enrollment. If your company uses the Volume Purchase Program (VPP), you can automatically install Company Portal app during enrollment without user Apple IDs. 
 - You want to lock the device until the Company Portal app installs.

> All The features mentioned above aren't supported in Apple Setup Assistant authentication methods. 

Reviewer: ```suggestion
All of the features mentioned in Option 1 aren't supported in Apple Setup Assistant authentication methods. 
```

> All of the features mentioned in Option 1 aren't supported in Apple Setup Assistant authentication methods. 

Reviewer: ```suggestion
None of the features mentioned in Option 1 are supported in Apple Setup Assistant authentication methods. 
```
